### DISCUSSION TITLE
When creating an EP Session, is there a way to pass the model directly in NPU Binary format instead of ONNX format?

### DISCUSSION THREAD

**Discussion by chansoos** [microsoft/onnxruntime#22887] [EP Q&A]
QNN or Openvino EP can pre-compile models described in ONNX format using their respective manufacturer tools. I'm wondering if it's possible to directly call the EP using these binaries, and if so, how can this be done?

**Comment by skottmckay** [comment]
@HectorSVC may be able to help with using a pre-compiled QNN model. Not sure about OpenVINO.
